lib/matplotlib/backends/backend_qt5.py

```diff
@@ -3,12 +3,13 @@
 import os
 import re
 import signal
+import socket
 import sys
 import traceback
 
 import matplotlib
 
-from matplotlib import backend_tools, cbook
+from matplotlib import backend_tools, cbook, pyplot
```

Comment:
I don't think we want a `pyplot` import leaking into backends, as that will cause many things to be set that are not necessarily desired.

Reply:
OK, I have removed it. It was only used to close the figure before app.quit(), and that was used to check that there are no more open windows. I have modified the test so it does not care how many figure windows are left.